app/Http/Controllers/ReportsController.php

```diff
@@ -1152,20 +1154,12 @@ public function sentAssetAcceptanceReminder(Request $request) : RedirectResponse
         }
 
         // Only send notification if assigned
-        if ($assetItem->assignedTo) {
+        if ($assetItem->assignedTo->email) {
```

Comment:
I think we might need to check that `$assetItem->assignedTo` is not null here before trying to get the property, no?

Reply:
done 👍